Releasenotes 5.2.1 (#586)
owncloud/docs-client-desktop: modules/ROOT/pages/appendices/release_notes.adoc

== Changelog for the Desktop App

ownCloud provides a full changelog with a summary and details for each release of the Desktop App. Click the following link to access it at {desktop-changelog-url}[GitHub].

== Desktop Sync Client 5.2.1

The ownCloud Desktop Client is now available in version 5.2.1! If you have an enterprise subscription, this is the first 5.x release available for branding. We strongly recommend updating the Desktop Sync Client to the latest available version so that you can access and experience a range of new features and security fixes. Users with an Enterprise subscription are entitled to receive dedicated support.

=== Enhanced Functionality

* Store proxy password securely
* Windows VFS: Prevent rename to ignored file name
* Change how all files deleted is handled
* Enable crash reporter in commandline client

> * Log http request when it is send

Reviewer: ```suggestion
* Log http request when it is sent
```

I would guess a bit about this, does it mean "Log the time that each http request is sent" or a general "Log http requests" feature (that log them somewhere in the client "when they are sent".

* --cmd argument added to the GUI client
* Improved reliability for persisting settings

=== Notable Bugfixes

ownCloud Desktop Client 5.2.1 also delivers several important technical improvements such as:

*	Client stuck in reconnecting
*	Fix url resolution for app provider
*	Fix crash on unhandled status code on rename check
*	Fix crash when keychain job takes longer than expected
*	Crash when accepting a notification
*	Fix crash on start-up when starting shell integration
*	Properly schedule the sync after an account was added
*	Don't start credentials save jobs during shutdown
*	Avoid duplicate notifications when selective sync is enabled
*	VFS Placeholders can now be replaced with folders
*	Authentication dialog no longer appears again and again
*	Possible deadlock during log setup
*	Do not create default sync root when loading accounts
*	Update capabilites and other info after connect
*	Account activity and crash after an account was removed
*	Crash during application shutdown
*	Hide hidden folders again in the selective sync view
*	Fix check if a file is a placeholder
*	Hydration state of file after a directory was replaced with a file

> *	Only syncronize after the server settings where refreshed

Reviewer: ```suggestion
*	Only syncronize after the server settings were refreshed
```

*	Consitently use the same icon for folders
*	Branding of folder status overlay
*	Creation of folder on the server

> *	Selective sync when not the remote root is synced

Reviewer: ```suggestion
*	Selective sync when other than the remote root is synced
```

Maybe this is a way to say it?

For an exhaustive list of bug fixes and changes, please refer to the full change log at {desktop-changelog-url}[GitHub]

=== Requirements

Please make sure that you meet the following requirements before updating to the latest version:

*	Windows 10, Version 1809+
*	Windows 11
*	macOS 10.15 Catalina
*	macOS 11 Big Sur
*	macOS 12 Monterey
*	macOS 13 Ventura
*	MacOS 14 Sonoma
*	Debian 11
*	Debian 12
*	Fedora 37
*	Fedora 38
*	Fedora 39
*	openSUSE Leap 15.4
*	openSUSE Leap 15.5
*	Ubuntu 22.04 LTS
*	Ubuntu 23.04
*	Ubuntu 23.10

> AppImage builds for other Linux distributions Here you can find the full requirements page.

Reviewer: Something is not right here?

Author: Hey Phil, what do you mean?

Reviewer 2: The uppercase Here probably.

=== Newly supported platforms

*	MacOS 14 Sonoma
*	Fedora 39
*	openSUSE Leap 15.5
*	Ubuntu 23.10

=== Deprecated

*	Windows 8
*	Windows 10, version 1709
*	macOS 10.13 High Sierra
*	macOS 10.14 Mojave

For Linux distributions, we support, if technically feasible, the latest two versions per platform and the latest LTS.

The following Linux versions are no longer supported:

*	Fedora 36
*	Ubuntu 22.10

=== Deprecation Announcements

Looking further, the following versions may no longer be supported in future releases:

*	macOS 10.15 Catalina
*	Fedora 37

=== Branding

If you're entitled to create branded versions of the ownCloud Desktop Client, please visit customer.owncloud.com to start the branding process for 5.2.1

> Updater Server 1.1.0

Reviewer: Should this be a heading or?
